### PR TITLE
🔐 Making auth relies on session cookies

### PR DESCRIPTION
**Pull request by abidon**

### CODE REVIEW

File: swiftpm/PortalAPI/Package.resolved [hunk 1]
```diff
@@ -108,24 +108,6 @@
         "revision" : "7e0948ac9e0743b854752e64aa0dbdef632f7922"
       }
     },
-    {
```
Comment (abidon) [inline swiftpm/PortalAPI/Package.resolved:111]: JWT are not used anymore

File: swiftpm/PortalAPI/Sources/Models/SignInCodeModel.swift [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import Fluent
+import Vapor
+
+public final class SignInTokenModel: Model, Authenticatable {
```
Comment (abidon) [inline swiftpm/PortalAPI/Sources/Models/SignInCodeModel.swift:4]: Sign In tokens are stored in database, meant to be used only once

File: swiftpm/PortalAPI/Sources/Models/UserModel.swift [hunk 3]
```diff
@@ -18,7 +18,6 @@ public final class UserModel: Model, Authenticatable {
     @Group(key: FieldKeys.mainEmail) public var mainEmail: UserEmailGroup
     
     @Siblings(through: OrganizationUserModel.self, from: \.$user, to: \.$organization) public var organizations: [OrganizationModel]
-    @Children(for: \.$user) public var sessionTokens: [SessionTokenModel]
```
Comment (abidon) [inline swiftpm/PortalAPI/Sources/Models/UserModel.swift:21]: No more JWT-based session tokens

File: package.json [hunk 4]
```diff
@@ -67,6 +67,7 @@
   },
   "devDependencies": {
     "@nrwl/js": "16.6.0",
+    "@nx-tools/nx-container": "^5.0.3",
```
Comment (Jeremy-Pichon) [inline package.json:70]: Update `develop-swift` from `develop` instead. This has been put back on develop branch already

File: swiftpm/PortalAPI/Sources/Modules/Auth/REST/AuthController.swift [hunk 5]
```diff
@@ -7,75 +7,88 @@ import Vapor
...
-        return SignInResponseContent(sessionToken: sessionToken, refreshToken: refreshToken)
+        guard let user else {
+            // this is supposedly unreachable
+            return HTTPStatus.internalServerError
+        }
+        
+        try await signInTokenRepo.deleteToken(signInToken, on: req.db)
+        req.session.authenticate(user)
+        
+        return HTTPStatus.noContent
```
Comment (Jeremy-Pichon) [inline swiftpm/PortalAPI/Sources/Modules/Auth/REST/AuthController.swift:83]: Could it return an object with like `{ initiateOnboarding: boolean }` instead? For web Onboarding purpose (`displayName + notificationOptIn`)

**Review by Jeremy-Pichon — APPROVED** [review]

**Pull request merged**